gitea/repo_file.go

```diff
@@ -13,3 +13,83 @@ import (
 func (c *Client) GetFile(user, repo, ref, tree string) ([]byte, error) {
 	return c.getResponse("GET", fmt.Sprintf("/repos/%s/%s/raw/%s/%s", user, repo, ref, tree), nil, nil)
 }
+
```

Comment:
Could you add `// Copyright 2019 The Gitea Authors. All rights reserved.` to header of this file?

Reply:
Done